### PR TITLE
Clone repo #17

### LINKED ISSUE #27: Feat: Add test for git clone function
Add unit test for the repoClone function in Repository.java

### PR DESCRIPTION
**Pull request by ekorre1001**
fixes #27

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -15,6 +15,9 @@ dependencies {
     implementation 'org.json:json:20211205'
     // https://mvnrepository.com/artifact/org.eclipse.jgit/org.eclipse.jgit
     implementation 'org.eclipse.jgit:org.eclipse.jgit:6.0.0.202111291000-r'
+    // https://mvnrepository.com/artifact/commons-io/commons-io
+    implementation 'commons-io:commons-io:2.6'
+
```
Comment (nolanderc) [inline build.gradle:20]: What advantage does this library have over the functions available in the Java standard library?

Comment (ekorre1001) [reply build.gradle:20]: It is used for deleting a directory. Do you know any easier solution?

Comment (nolanderc) [reply build.gradle:20]: @mantaur used the `File.delete` method here:
https://github.com/KTH-Software-Engineering-DD2480/ci/blob/f2714e502f9bee23f3b2b4931d7d8f3d90ef6b1e/src/test/java/ci/PersistentLogsTest.java#L13-L22

we could move

Comment (ekorre1001) [reply build.gradle:20]: Alright, let me try!

Comment (ekorre1001) [reply build.gradle:20]: Running the same function, only the files inside the folder are deleted not the folder itself.

Comment (nolanderc) [reply build.gradle:20]: Huh, strange... Well, keep the working version then :)

**Pull request merged**